var/spack/repos/builtin/packages/libjpeg-turbo/package.py

```diff
@@ -58,9 +60,15 @@ class LibjpegTurbo(CMakePackage, AutotoolsPackage):
         default="cmake",
     )
 
-    variant("shared", default=True, description="Build shared libs")
-    variant("static", default=True, description="Build static libs")
+    variant(
+        "libs",
+        default=("shared", "static"),
+        values=("shared", "static"),
+        multi=True,
+        description="Build shared libs, static libs, or both",
+    )
     variant("jpeg8", default=False, description="Emulate libjpeg v8 API/ABI")
+    variant("pic", default=True, description="Enable position independent code")
```

Comment:
Should we activate this one only when `spec.satisfies("libs=static")`?

Reply:
I think so yes. Would it make sense to also have a conflict between ~pic and libs=shared to additionally avoid the `~pic libs=shared,static` case?

Reply:
No, that would be redundant. Just:
```suggestion
    variant("pic", default=True, description="Enable position independent code", when="libs=static")
```